cfe_internal/enterprise/CFE_knowledge.cf

```diff
@@ -357,3 +357,20 @@ body depth_search cfe_internal_docroot_application_perms
       depth => "inf";
       exclude_dirs => { "logs" };
 }
+
+############################################################################
+
+body perms state_dir_perms
+{
+      mode   => "0600";
+      owners => { "root" };
+
+      freebsd|openbsd|netbsd|darwin::
+        groups => { "wheel" };
+
+      aix::
+        groups => { "system" };
+
+      !(freebsd|openbsd|netbsd|darwin|aix)::
+        groups => { "root" };
```

Comment:
Windows?

Reply:
Right. This wasn't handled before so must work as-is? How are windows perms handled? I can check/test.